torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -25,14 +25,24 @@ inline std::vector<int64_t> bufferSizes(const T& t) {
 }
 using ArgNone = c10::monostate;
 using BufList = std::vector<tensorexpr::BufHandle>;
+using IntList = std::vector<int64_t>;
```

Comment:
`IntList` doesn't seem to be used anywhere.

Reply:
This is used in some other lowerings - I just added all the lists here.